docs/getting-started/improvements.md

```diff
@@ -1 +1,19 @@
 # Improvements
+
+The following section contains suggestions for improving the application and may be extended much further.
+
+## Tiling of garden images to increase loading speed
+
+Administrators of a companies or gardens have the ability to upload a (self-taken) picture of a garden using
+the web interface. The image can be displayed in the dashboard-, beds- or plants-view in the web interface,
+replacing the satellite image at the exact location of the garden, providing a much more detailed
+view of the beds and plants.
+
+User images can have a large filesize, if taken with a high quality camera. This leads to long loading times
```

Comment:
```suggestion
User images can have a large filesize, if taken with a drone This leads to long loading times
```

Reply:
@pbrozi @jarkenau For the **file size** (not filesize) it does not matter whether it was made with a good camera or a drone.

Reply:
Large ladder... 🥲 
The whole **if part** just doesn't make sense like that.